src/python/scripts/dx-jobutil-get-identity-token

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2013-2024 DNAnexus, Inc.
+#
+# This file is part of dx-toolkit (DNAnexus platform client libraries).
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#   use this file except in compliance with the License. You may obtain a copy
+#   of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+import dxpy
+import os, argparse, json, sys
+
+parser = argparse.ArgumentParser(
+    description="Creates a new job to run the named function with the specified input.  If successful, prints the ID of the new job."
+)
+parser.add_argument("--aud", help="Audience URI the JWT is intended for", required=True)
+parser.add_argument(
+    "--subject_claims",
+    action="append",
+    metavar="<subject_claims>",
+    help="Defines the subject claims to be validated by the cloud provider",
+)
+
+args = parser.parse_args()
+job_get_identity_token_input = {}
+
+# Parse out audience and subject_claims from args
+if args.aud is not None:
+    job_get_identity_token_input["audience"] = args.aud
+if args.subject_claims is not None:
+    # Iterate over subject_claims and flatten them into a single list
+    subject_claims = []
+    for subject_claim in args.subject_claims:
+        subject_claims.extend(subject_claim.split(","))
+    job_get_identity_token_input["subjectClaims"] = subject_claims
```

Comment:
shouldn't be the request field name `subject_claims`?

Reply:
on the javascript side everything is `camelCase` but python is `snake case`. We decided as a team to have python keep python standards and the api adhere to js standards